### ISSUE TITLE
Can't use security keys to login into Apple account.

### ISSUE THREAD

**Issue by afonsofrancof** [Mastermindzh/tidal-hifi#248]
It says unsupported browser every time I try to login with apple using my security key.

There seems to be a way to intercept the protocol, but I'm not sure if it can be implemented here.
https://github.com/electron/electron/issues/24573

**Comment by Mar0xy** [comment]
It can not be implemented cause as the comment says "this will break all http requests but localhost"

even firefox doesn't properly support it yet and I don't know why so many are already jumping on the train of using them when it's obv that proper support isn't there yet even if big tech keeps marketing about it.

**Comment by Mastermindzh** [comment]
Agreed. Unless it becomes part of the base functionality set of the browser implementing it is out of the question honestly.
